schunk_sdh/msg/TemperatureArray.msg

```diff
@@ -0,0 +1,2 @@
+Header header
+Temperature[] temperature_list
```

Comment:
I am happy with either way; while being similar to ``sensor_msgs/JointState`` is the more standard approach for message types, I like the motivation with iterators though so let's proceed with that.